### PR TITLE
Add commands for creating / updating Harbors

### PR DESCRIPTION
**Pull request by igordavydsson**
Refs #7

### CODE REVIEW

File: harbors/management/commands/create_harbors_from_json.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument('--file', action='store', dest='file',
+                            help="Path to JSON file with harbors' data")
+
+    def handle(self, **options):
+        json_filepath = options['file']
+        if not json_filepath:
+            self.stderr.write('No path to JSON file provided')
```
Comment (tuomas777) [inline harbors/management/commands/create_harbors_from_json.py]: Probably better to raise [CommandError](https://docs.djangoproject.com/en/2.1/howto/custom-management-commands/#django.core.management.CommandError) here?

File: harbors/management/commands/create_harbors_from_json.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+        number_of_created_harbors = 0
+        number_of_modified_harbors = 0
+
+        with open(json_filepath, 'r') as json_file:
+            harbors_dict = json.load(json_file)
+            for harbor_name, harbor_data in harbors_dict.items():
+                harbor, created = Harbor.objects.get_or_create(identifier=slugify(harbor_name))
```
Comment (tuomas777) [inline harbors/management/commands/create_harbors_from_json.py]: Could use [update_or_create()](https://docs.djangoproject.com/en/2.1/ref/models/querysets/#update-or-create) here.

File: harbors/management/commands/update_harbors_from_servicemap.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+            harbor.zip_code = sm_info['address_zip']
+            harbor.www_url = sm_info['www']['fi']  # only FI for now
+            harbor.email = sm_info['email']
+            harbor.phone = sm_info['phone']
+
+            harbor.municipality = Municipality.objects.get(id=sm_info['municipality'])
+
+            latitude = sm_info['location']['coordinates'][1]
+            longitude = sm_info['location']['coordinates'][0]
```
Comment (tuomas777) [inline harbors/management/commands/update_harbors_from_servicemap.py]: Fancy people would do `longitude, latitude = sm_info['location']['coordinates']`.

File: harbors/management/commands/update_harbors_from_servicemap.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            harbor.location = Point(longitude, latitude, srid=settings.DEFAULT_SRID)
+
+            harbor.save()
```
Comment (tuomas777) [inline harbors/management/commands/update_harbors_from_servicemap.py]: Is there some reason why there cannot be just one `harbor.save()` at the end of the loop?

Comment (igordavydsson) [reply harbors/management/commands/update_harbors_from_servicemap.py]: Not sure, that you can set properly translated fields without saving each language... This `harbor.save()` line might be deleted though? if we save the object pretty soon when handling translated field?

Comment (tuomas777) [reply harbors/management/commands/update_harbors_from_servicemap.py]: Ok, sounds a bit stupid, but might be, I don't know Parler that well :) If that is the case, I would keep this save as well to make sure the object is saved even when there are no translations in the upcoming translated fields (probably not possible in practice but still).

**Pull request merged**